Update dynamic-theme-fixes.config for YouTube (youtube.com)

erkingoksuu opened the pull request:
-Fixed some text color in light mode such as channel name or secondary section 'yt-related-chip-cloud-renderer' "STYLE_DEFAULT" color

-Fixed Join button colors in both dark and light mode

-Fixed Subscribe button and other buttons as like, dislike, share... colors in both dark and light mode

-Fixed Sort button background colors in both dark and light mode

-Fixed all comments background colors in both dark and light mode

-Some additional small color fix

erkingoksuu commented:
-Fixed some text color in light mode such as channel name or secondary section 'yt-related-chip-cloud-renderer' "STYLE_DEFAULT" color

-Fixed Join button colors in both dark and light mode

-Fixed Subscribe button and other buttons as like, dislike, share... colors in both dark and light mode

-Fixed Sort button background colors in both dark and light mode

-Fixed all comments background colors in both dark and light mode

-Some additional small color fix

Myshor commented:
As I use detecting of dark mode in Dark Reader I even don't have Dark Reader enabled for Youtube.

Can you show the screen examples what was fixed?
It would really help checking how fixes work when Dark mode is set on youtube, when youtube is set to light mode, when Dark Reader is set to dark scheme, when Dark Reader is set to light scheme.

erkingoksuu commented:
I encounter the bug when youtube is dark mode,

All Dark Mode, Not Fixed:

![image](https://github.com/user-attachments/assets/9de05f38-bcb7-43c4-8f74-486b766ba991)

All  Dark Mode, Fixed:

![image](https://github.com/user-attachments/assets/ba25f562-a2ea-41e5-8bb6-653ea9320b96)

Youtube Dark Mode, Dark Reader Light Mode, Not Fixed:

![image](https://github.com/user-attachments/assets/0f83b7f8-fe1e-4fd2-bc80-78b117b4d3a0)

Youtube Dark Mode, Dark Reader Light Mode, Fixed:

![image](https://github.com/user-attachments/assets/3c6e2bc4-3700-4fcd-9792-ef64e9363288)

if you have additional questions pls ask
